multi_objective/requirements.txt

```diff
@@ -1,5 +1,5 @@
-botorch
+botorch>=0.4.0,< 0.8.0
 fvcore
 torch
 torchaudio
-torchvision
+torchvision
```

Comment:
I don't think this line has any changes. Do you have any ideas?

Reply:
You' right. This part should not be changed.